modules/gitops-creating-aggregated-cluster-roles.adoc

```diff
@@ -0,0 +1,140 @@
+
+The following is an example definition of an aggregated cluster role:
+
+.Example aggregated cluster role YAML
+[source,yaml]
+----
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: <argocd_name>-<argocd_namespace>-argocd-application-controller # <1>
+  annotations:
+    rbac.authorization.kubernetes.io/autoupdate: 'true'
+aggregationRule: # <2>
+  clusterRoleSelectors:
+    - matchLabels: 
+        app.kubernetes.io/managed-by: <argocd_namespace>
+        argocd/aggregate-to-controller: 'true'
+rules: [] # <3>
+----
+<1> The name of the aggregated cluster role.
+<2> The predefined list of labels indicates that the aggregated cluster role can inherit permissions from the other user-defined cluster roles.
+<3> No predefined permissions are set. However, when the Operator immediately creates a `<argocd_name>-<argocd_namespace>-argocd-application-controller-view` cluster role, the corresponding predefined `view` permissions are added into the aggregated cluster role.
+
+The following is an example definition of a `<argocd_name>-<argocd_namespace>-argocd-application-controller-view` cluster role:
+
+.Example cluster role YAML
+[source,yaml]
+----
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: <argocd_name>-<argocd_namespace>-argocd-application-controller-view # <1>
+  labels: # <2>
+    app.kubernetes.io/managed-by: <argocd_namespace>
+    argocd/aggregate-to-controller: 'true'
+rules: # <3>
+  - verbs:
+      - get
+      - list
+      - watch
+    apiGroups:
+      - '*'
+    resources:
+      - '*'
+  - verbs:
+      - get
+      - list
+    nonResourceURLs:
+      - '*'
+----
+<1> The name of the `view` cluster role.
+<2> The labels match the predefined list of an existing aggregated cluster role.
+<3> The predefined `view` permissions.
++
+[NOTE]
+====
+These permissions are added into the existing aggregated cluster role.
+====
+
+The following is an example definition of an `<argocd_name>-<argocd_namespace>-argocd-application-controller-admin` cluster role:
+
+.Example cluster role YAML
+[source,yaml]
+----
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: <argocd_name>-<argocd_namespace>-argocd-application-controller-admin # <1>
+  labels: # <2>
+    app.kubernetes.io/managed-by: <argocd_namespace>
+    argocd/aggregate-to-controller: 'true'
+aggregationRule: # <3>
+  clusterRoleSelectors:
+    - matchLabels: 
+        app.kubernetes.io/managed-by: <argocd_namespace>
+        argocd/aggregate-to-admin: 'true'
+rules: [] # <4>
+----
+<1> The name of the `admin` cluster role.
+<2> The labels match the predefined list an existing aggregated cluster role.
```

Comment:
Suggestion: 

From 

The labels match the predefined list an existing aggregated cluster role.

To

The labels match the predefined list **of** an existing aggregated cluster role.